Library/Homebrew/formula_auditor.rb

```diff
@@ -590,6 +590,28 @@ def audit_bottle_spec
       new_formula_problem "New formulae in homebrew/core should not have a `bottle do` block"
     end
 
+    def audit_wayback_url
+      return unless @core_tap
+      return if formula.deprecated? || formula.disabled?
+
+      regex = %r{^https?://web\.archive\.org}
+      if regex.match?(formula.stable.url)
+        problem "Formula with a Wayback URL should be deprecated with `:repo_removed`"
+        return
```

Comment:
Might not want to `return` here in case this applies in all 3 cases?

Reply:
Yeah, I wasn't really sure what to do there.